talk/owt/sdk/base/peerconnectionchannel.cc

```diff
@@ -28,6 +28,7 @@ bool PeerConnectionChannel::InitializePeerConnection() {
   audio_transceiver_direction_ = webrtc::RtpTransceiverDirection::kSendRecv;
   video_transceiver_direction_ = webrtc::RtpTransceiverDirection::kSendRecv;
   configuration_.enable_dtls_srtp = true;
+  configuration_.crypto_options->srtp.enable_gcm_crypto_suites = true;
```

Comment:
This change doesn't related to WebTransport.

Reply:
Removed this and will submit a seperate PR for this.